docs/workflows/logic/mutate.md

```diff
@@ -1,16 +1,20 @@
+flowclass: Cloud
+flowclass: Edge 1.0.0
+description: Learn more about the Losant Mutate Node.
+
 # Mutate Node
 
-The mutate node allows a workflow to perform arbitrary changes to the structure of the payload object.
+The Mutate Node allows a workflow to perform arbitrary changes to the structure of the payload object.
 
 ![Mutate Node](/images/workflows/logic/mutate-node.png "Mutate Node")
 
 ## Configuration
 
-Mutate nodes are configured by creating "rules", of which there are 4 types - "Set", "Remove", "Copy", and "Move". A mutate node can have as many rules as is needed, and the rules are executed in order from the first to last, which means subsequent rules can assume that the changes from previous rules have taken place.
+Mutate Nodes are configured by creating "rules", of which there are 4 types - "Set", "Remove", "Copy", and "Move". A Mutate Node can have as many rules as is needed, and the rules are executed in order from the first to last, which means subsequent rules can assume that the changes from previous rules have taken place.
```

Comment:
this figure 4 should be spelled out to the word four

Reply:
👍